### DISCUSSION TITLE
Repository Transfer Improvement Suggestion

### DISCUSSION THREAD

**Discussion by cqustlym** [community/community#40271] [Repositories]
### Select Topic Area

Product Feedback

### Body

Greetings! I want to make a suggestion about the repository transfer notification system. 

Here is my experience: My partner today transferred his repository to me because he will be busy later. But I did not receive any response. He said he did the transfer operation. So I just wait and refresh the github's webpage. Nothing happened. Later I have to send an important email to my boss so I opened my gmail. Guess what? I found an email from Github. It said I have a repository transfer. I must click it to accept it. WoW! That's where you are!

It will be nice if Github annonce or notifies the user when others are making a repository transfer. The user now does not receive the message at the first time. If we can receive the message just from the Github website not from the email the efficiency will be much higher in my opinion.

**Comment by ossanna16** [comment]
Hi there @cqustlym and welcome to our community! Thank you for the suggestion!
